kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-on-pv/wdt/wdt_model_configured.yaml

```diff
@@ -2,6 +2,11 @@ domainInfo:
     AdminUserName: '@@FILE:/weblogic-operator/secrets/username@@'
     AdminPassword: '@@FILE:/weblogic-operator/secrets/password@@'
     ServerStartMode: prod
+    RCUDbInfo:
```

Comment:
Just a style question... I see that below you have a space between the key name and the colon.  Why is this?  If it's not meaningful, then we should match the style of the rest of the file with no such space.